### ISSUE TITLE
using lighter backbone

### ISSUE THREAD

**Issue by athulvingt** [CatchZeng/object-detection-api#2]
is it possible to train with lighter models like resenet18, resnet50?

**Comment by CatchZeng** [comment]
Select the appropriate model from [Model Zoo](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/tf2_detection_zoo.md). See https://makeoptim.com/en/deep-learning/yiai-image-segmentation#download-the-pre-trained-model

**Comment by athulvingt** [comment]
In TF model Zoo, there is only one Mask rcnn pretrained model, is it possible to change the backbone to lighter version of resnet like resnet18 or resnet 50?

**Comment by CatchZeng** [comment]
TF model Zoo does not provide these pre-trained models, you can train one yourself, or new an issue at https://github.com/tensorflow/models/issues.
